src/test/objectstore/store_test.cc

```diff
@@ -1317,6 +1317,99 @@ TEST_P(StoreTest, SimpleObjectTest) {
 
 #if defined(WITH_BLUESTORE)
 
+TEST_P(StoreTestSpecificAUSize, ReproBug41901Test) {
+  if(string(GetParam()) != "bluestore")
+    return;
+  SetVal(g_conf(), "bluestore_debug_enforce_settings", "hdd");
+  g_conf().apply_changes(nullptr);
```

Comment:
@neha-ojha So, just drop the following lines from the commit?

```
  SetVal(g_conf(), "bluestore_debug_enforce_settings", "hdd");
  g_conf().apply_changes(nullptr);
```

Reply:
@smithfarm don't think that's enough, @ifed01 thoughts?

Reply:
@neha-ojha, @smithfarm - no, that's not enough - test case will fail when running from SSD. I can see two options here - remove the test case itself or backport commit(s) introducing "bluestore_debug_enforce_settings" setting. Personally I think removing test case in this backport is OK.